Assets/MixedRealityToolkit/UX/Scripts/Progress/ObjectDisplayScript.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObjectDisplayScript : MonoBehaviour
+{
+    [Header("How fast does object rotate?")]
+    public float rotationIncrement;
```

Comment:
We should probably encapsulate these fields with properties if they need to be accessed by an external class and make these private and serialize the fields.

Reply:
Also, use PascalCasing for public and camelCasing for private

Reply:
(Goes for all classes in this PR.